projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1128,6 +1129,25 @@ public function register_post_meta() {
 			register_meta( 'post', $this->POST_DONE . 'all', $already_shared_flag_args );
 			register_meta( 'post', self::POST_JETPACK_SOCIAL_OPTIONS, $jetpack_social_options_args );
 		}
+
+		add_filter(
+			'default_post_metadata',
+			function ( $value, $object_id, $meta_key ) {
```

Comment:
It might be better if this is a method on the class, so it's easier for someone to remove, if needs be.